### PR TITLE
bar_11

### LINKED ISSUE #117: bar_11 example
Description: Paneling 4 bar plots with a shared legend for colors

Script URL: https://www.ncl.ucar.edu/Applications/Scripts/bar_11.ncl
Plot URL: https://www.ncl.ucar.edu/Applications/Images/bar_11_lg.png
Category: Bar Charts

### PR DESCRIPTION
**Pull request by hCraker**
Closes #117 
image

### CODE REVIEW

File: Plots/Bar/NCL_bar_11.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/bar_11.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/bar_11_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import matplotlib.pyplot as plt
+import numpy as np
+
+from geocat.viz import cmaps as gvcmaps
```
Comment (michaelavs) [inline Plots/Bar/NCL_bar_11.py:22]: This is imported and unused

File: Plots/Bar/NCL_bar_11.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+"""
+NCL_bar_11.py
+===============
+This script illustrates the following concepts:
```
Comment (clairefio) [inline Plots/Bar/NCL_bar_11.py:4]: Do "changing the aspect ratio of a bar plot" and "setting the min/max value of the X and Y axis in a bar plot" still apply to the python version?

Comment (hCraker) [reply Plots/Bar/NCL_bar_11.py:4]: The first one no longer applies but the second one does. I set the limits of the axes.

**Comment by hCraker** [comment]
Updated plot after adjusting the figure size, gridspec `wspace`, and font sizes.
image

**Pull request merged**